table-attributes.md

```diff
@@ -19,50 +19,52 @@ The table attribute is in the form of `key=value`. Multiple attributes are separ
 + Set attributes for a table or partition:
 
     ```sql
-    alter table t [partition p ]attributes[=]'key=value[, key1=value1...]';
+    ALTER TABLE t [PARTITION p ]ATTRIBUTES[=]'key=value[, key1=value1...]';
     ```
 
 + Reset attributes for a table or partition:
 
     ```sql
-    alter table t [partition p ]attributes[=]default;
+    ALTER TABLE t [PARTITION p ]ATTRIBUTES[=]default;
     ```
 
 + See the attributes of all tables and partitions:
 
     ```sql
-    select * from information_schema.attributes;
+    SELECT * FROM information_schema.attributes;
     ```
 
 + See the attribute configured to a table or partition:
 
     ```sql
-    select * from information_schema.attributes where id='schema/t[/p]';
+    SELECT * FROM information_schema.attributes WHERE id='schema/t[/p]';
     ```
 
 + See all tables and partitions that have a specific attribute:
 
     ```sql
-    select * from information_schema.attributes where attributes like '%key%';
+    SELECT * FROM information_schema.attributes WHERE attributes LIKE '%key%';
     ```
 
 ## Attribute override rules
 
 The attribute configured to a table takes effect on all partitions of the table. However, there is one exception: If the table and partition are configured with the same attribute but different attribute values, the partition attribute overrides the table attribute. For example, suppose that the table `t` is configured with the `key=value` attribute, and the partition `p` is configured with `key=value1`.
 
 ```sql
-alter table t attributes[=]'key=value';
-alter table t partition p attributes[=]'key=value1';
+ALTER TABLE t ATTRIBUTES[=]'key=value';
+ALTER TABLE t PARTITION p ATTRIBUTES[=]'key=value1';
 ```
 
 In this case, `key=value1` is the attribute that actually takes effect on the `p1` partition.
 
-## Control the Region merge bahavior using table attributes
+## Control the Region merge behavior using table attributes
 
 ### User scenarios
 
 If there is a write hotspot or read hotspot, you can use table attributes to control the Region merge behavior. You can first add the `merge_option` attribute to a table or partition and then set its value to `deny`. The two scenarios are as follows.
 
+This is only part of how to deal with hotspots. For more information, refer to [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md). 
```

Comment:
@dveeden how about moving this part above "### Usage" and changing it as a "Note"?

Reply:
Done.